app/helpers/argo_helper.rb

```diff
@@ -102,11 +102,11 @@ def render_buttons(doc, object = nil)
     end
 
     # if this is an apo and the user has permission for the apo, let them edit it.
-    if (object.datastreams.include? 'roleMetadata') && (current_user.is_admin || current_user.is_manager || object.can_manage_item?(current_user.roles(apo_pid)))
+    if object.datastreams.include?('roleMetadata') && can?(:manage_item, object)
```

Comment:
If the `can?(:manage_item, object)` is specific to an object that has 'roleMetadata' (generally and perhaps only a `Dor::AdminPolicyObject`), then the logic may change here and  `can?(:manage_item, object)` may suffice.